spec/client/user_client_spec.rb

```diff
@@ -295,6 +295,9 @@
     context 'when user_id is defined' do
       let(:user_id) { '95985b21-d884-4bbd-b852-cb8cd365afc2' }
 
+      # temporarily disabled because this test needs refactor!
+      before { skip }
```

Comment:
Refactoring is not a valid reason to skip tests.
I agree the code is not perfect, but if the tests are skipped, the PR is not complete.